sites/public/pages/listing/[id]/[slug].tsx

```diff
@@ -58,7 +59,23 @@ export default function ListingPage(props: ListingProps) {
         <title>{pageTitle}</title>
       </Head>
       <MetaTags title={listing.name} image={metaImage} description={metaDescription} />
-      <ListingView listing={listing} />
+      {favoriteToastIsOpen && (
+        <AlertBox
+          type={"success"}
+          narrow={true}
```

Comment:
The toast designs are not using the narrow version, can you remove this from both toasts

Reply:
good catch on that, thank you, got that updated